### DISCUSSION TITLE
[30 Giugno 2022] - Cache

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#10] [Cache]
<img width="948" alt="Esame" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/7ef4b98c-eb47-401a-baa5-6a625de56816">

---
>**[Altre Soluzioni](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/discussions/61)**

**Comment by notedo** [comment]
![ESERCIZI - Architettura-76](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/61e24c5e-68d2-4292-a417-fcb7c4e71c08)

**Comment by rimaout** [comment]
### 1) 
![IMG_3142](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/41346035-98a0-4555-a3d3-2bf18ba0157f)

### 2) 
<img width="1062" alt="1052384B-8EDA-4B18-B31D-301D05E4C080" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/54a72fa9-bdd3-4e19-959f-b019d8a96fa1">

### 3)
<img width="1010" alt="5DAED07D-AE29-4099-B063-E6F349374FFE" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/2e8c869b-6e2f-455c-927a-6ef42993e69b">


### 4)
<img width="785" alt="08B5DAE6-311B-467B-A5A9-0711DED55179" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/f81d9c57-efeb-4c03-b728-4227b98c2a16">
